src/unix/linux-core.c

```diff
@@ -278,33 +280,46 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
   if (statfile_fp == NULL)
     return UV__ERR(errno);
 
-  err = uv__cpu_num(statfile_fp, &numcpus);
-  if (err < 0)
-    goto out;
+  for (i = 0; i < 3; i += 1) {
```

Comment:
style nit: the magic number here needs a comment about the purpose

Reply:
@vtjnash thank you for the suggestions. updated with a static const variable `max_retry` to reflect their meaning.